OpenNefia.Content/Locale/Funcs/GermanBuiltins.cs

```diff
@@ -0,0 +1,93 @@
+﻿using OpenNefia.Content.DisplayName;
+using OpenNefia.Content.GameObjects;
+using OpenNefia.Core.Game;
+using OpenNefia.Core.GameObjects;
+using OpenNefia.Core.IoC;
+using OpenNefia.Core.Locale;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OpenNefia.Content.Locale.Funcs
+{
+    [RegisterLocaleFunctions("de_DE")]
+    public static class GermanBuiltins
+    {
+        public const string BaseArticle = "ein";
+        public const string GenderAttributeName = "Gender";
+        public const string PluralAttributeName = "Plural";
+
+        public const string PluralNameAttributeName = "PluralRule";
+        public const string PluralNameAlways = "always";
+
+        public const string GenderNameMale = "male";
+        public const string GenderNameFemale = "female";
+        public const string GenderNameNeutral = "neutral";
+
+        public const string AdjectiveAttributeName = "Adjective";
+        public const string AdjectiveDirectAttributeName = "AdjectiveDirect";
+
+        public const string ArticleSuffixFemale = "e";
+        public const string ArticleSuffixMale = "en";
+        /// <summary>
```

Comment:
Nitpick: indentation between function/constants.

Reply:
changed in 0845470